core/models_template.py

```diff
@@ -0,0 +1,314 @@
+    aid_type_code = models.IntegerField()
+    aid_type_name = models.TextField()
+    flows = models.SmallIntegerField()
+    fund_flows = models.TextField()
+    amount_type_code = models.CharField(max_length=1)
+    amount_type_name = models.TextField()
+    time = models.SmallIntegerField()
+    year = models.SmallIntegerField()
+    value = models.DecimalField(max_digits=65535, decimal_places=65535, blank=True, null=True)
+    flags = models.TextField(blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = 'dac1_current'
+
+
+class Dac2ACurrent(models.Model):
+    row_id = models.AutoField(primary_key=True)
+    recipient_code = models.SmallIntegerField()
+    recipient_name = models.TextField()
+    donor_code = models.SmallIntegerField()
+    donor_name = models.TextField()
+    part_code = models.IntegerField()
+    part_name = models.TextField()
+    aid_type_code = models.SmallIntegerField()
+    aid_type_name = models.TextField()
+    data_type = models.CharField(max_length=1)
+    amount_type = models.TextField()
+    time = models.SmallIntegerField()
+    year = models.SmallIntegerField()
+    value = models.DecimalField(max_digits=65535, decimal_places=65535, blank=True, null=True)
+    flags = models.TextField(blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = 'dac2a_current'
+
+
+class Dac2BCurrent(models.Model):
+    row_id = models.AutoField(primary_key=True)
+    recipient_code = models.SmallIntegerField()
+    recipient_name = models.TextField()
+    donor_code = models.SmallIntegerField()
+    donor_name = models.TextField()
+    part_code = models.IntegerField()
+    part_name = models.TextField()
+    aid_type_code = models.SmallIntegerField()
+    aid_type_name = models.TextField()
+    data_type = models.CharField(max_length=1)
+    amount_type = models.TextField()
+    time = models.SmallIntegerField()
+    year = models.SmallIntegerField()
+    value = models.DecimalField(max_digits=65535, decimal_places=65535, blank=True, null=True)
+    flags = models.TextField(blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = 'dac2b_current'
+
+
+class Dac5Current(models.Model):
+    row_id = models.AutoField(primary_key=True)
+    donor_code = models.SmallIntegerField()
+    donor_name = models.TextField()
+    sector_code = models.SmallIntegerField()
+    sector_name = models.TextField()
+    aid_type_code = models.IntegerField()
+    aid_type_name = models.TextField()
+    amount_type_code = models.CharField(max_length=1)
+    amount_type_name = models.TextField()
+    time = models.SmallIntegerField()
+    year = models.SmallIntegerField()
+    value = models.DecimalField(max_digits=65535, decimal_places=65535, blank=True, null=True)
+    flags = models.TextField(blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = 'dac5_current'
+
+
+class Operation(models.Model):
```

Comment:
@Napho can we move these into `models.py` and make them managed by Django's ORM directly? Even after following the DB setup steps, we cannot add them to admin.py without the following error message: 
![image](https://user-images.githubusercontent.com/2836840/53586607-89678580-3b56-11e9-8ff8-dc1014a87ca3.png)

Reply:
Cool, for ease of use i wiill do that, though i think the query is failing because its targeting the public schema